docs/reference_manual/core_api.rst

```diff
@@ -63,3 +64,7 @@ This chapter describes all public methods in the SiliconCompiler core Python API
 
 .. automodule:: siliconcompiler.core
     :members:
+
+.. automodule:: siliconcompiler.use
```

Comment:
It this needed? if you add it above

Reply:
This refers to the modules in `siliconcompiler/use.py`, not `Chip.use`